packaging/helm-charts/helm3/strimzi-drain-cleaner/templates/_helpers.tpl

```diff
@@ -51,6 +51,13 @@ Selector labels
 app: {{ include "strimzi-drain-cleaner.name" . }}
 {{- end }}
 
+{{/*
+Annotations
+*/}}
+{{- define "strimzi-drain-cleaner.annotations" -}}
+app: {{ include "strimzi-drain-cleaner.name" . }}
```

Comment:
Why should we have this annotation there? I would say it should be empty by default.

Reply:
I have submitted a new commit with this removed as you are right.